packages/react-native-audio-api/common/cpp/audioapi/jsi/AudioArrayBuffer.cpp

```diff
@@ -0,0 +1,16 @@
+#include <audioapi/jsi/AudioArrayBuffer.h>
+
+namespace audioapi {
+
+AudioArrayBuffer::AudioArrayBuffer(float *data, size_t size)
+    : data_(data), size_(size) {}
+
+size_t AudioArrayBuffer::size() const {
+  return size_;
+}
+
+uint8_t *AudioArrayBuffer::data() {
+  return reinterpret_cast<uint8_t *>(data_);
```

Comment:
Another option would be to use `reinterpret_cast` in constructor (instead of in `data()` method) and store `data_` as `uint8_t *` instead of `float *`. Which option do you think is better?